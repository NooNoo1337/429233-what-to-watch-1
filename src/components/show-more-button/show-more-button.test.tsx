import * as React from 'react';
import * as renderer from 'react-test-renderer';

// Components
import ShowMoreButton from './show-more-button';

const mockFilmCounter = 25;
const mockFilmsToShow = 20;

describe(`ShowMoreButtonComponent`, () => {
  it(`should render correctly`, () => {
    const tree = renderer
      .create(
        <ShowMoreButton
          onFilmsLimitChange={() => {}}
          filmsCounter={mockFilmCounter}
          filmsToShow={mockFilmsToShow}
        />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
