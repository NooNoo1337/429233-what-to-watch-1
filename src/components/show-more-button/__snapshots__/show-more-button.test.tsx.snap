// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ShowMoreButtonComponent should render correctly 1`] = `
<div
  className="catalog__more"
  style={
    Object {
      "display": "block",
    }
  }
>
  <button
    className="catalog__button"
    onClick={[Function]}
    type="button"
  >
    Show more
  </button>
</div>
`;
